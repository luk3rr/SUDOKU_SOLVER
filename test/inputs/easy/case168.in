030064700 700903002 008200000 203000810 900000570 085000200 300700000 000400006 040006900 
