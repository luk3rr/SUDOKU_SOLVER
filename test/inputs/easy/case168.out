Solving the following grid:
0 3 0 | 0 6 4 | 7 0 0 
7 0 0 | 9 0 3 | 0 0 2 
0 0 8 | 2 0 0 | 0 0 0 
------+-------+------
2 0 3 | 0 0 0 | 8 1 0 
9 0 0 | 0 0 0 | 5 7 0 
0 8 5 | 0 0 0 | 2 0 0 
------+-------+------
3 0 0 | 7 0 0 | 0 0 0 
0 0 0 | 4 0 0 | 0 0 6 
0 4 0 | 0 0 6 | 9 0 0 


Solution found :')

1 3 2 | 8 6 4 | 7 9 5 
7 5 4 | 9 1 3 | 6 8 2 
6 9 8 | 2 5 7 | 3 4 1 
------+-------+------
2 7 3 | 6 4 5 | 8 1 9 
9 1 6 | 3 8 2 | 5 7 4 
4 8 5 | 1 7 9 | 2 6 3 
------+-------+------
3 6 9 | 7 2 1 | 4 5 8 
5 2 7 | 4 9 8 | 1 3 6 
8 4 1 | 5 3 6 | 9 2 7 

Algorithm: BFS
Total time: 62 ms
Total expanded states: 10847
